shared/models/videos/channel-sync/video-channel-sync.model.ts

```diff
@@ -0,0 +1,14 @@
+import { VideoChannelSummary } from '../channel/video-channel.model'
+import { VideoConstant } from '../video-constant.model'
+import { VideoChannelSyncState } from './video-channel-sync-state.enum'
+
+export interface VideoChannelSync {
+  id: number
+
+  externalChannelUrl: string
+
+  createdAt: string
+  channel: VideoChannelSummary
+  state: VideoConstant<VideoChannelSyncState>
+  lastSyncAt?: string
```

Comment:
`lastSyncAt` is `string` or `null`, but I don't think it can be missing or `undefined`